source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,90 +1,84 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Fall back to the static mirrors
+title: GOV.UK CDN static mirrors
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-We maintain a static copy of most of the site, which gets used by the content delivery
-network (CDN) whenever origin (the application server) times out or serves an error
-response.
+GOV.UK uses the [Fastly content delivery network (CDN)](/manual/cdn.html) to handle most requests to [GOV.UK Origin](/manual/architecture-shallow-dive.html#a-user-visits-the-gov-uk-homepage). However, there are 2 scenarios where the user requests will not get to Fastly.
 
-This process is handled by our CDN config and is entirely transparent to us and
-our users. It happens multiple times a day, for lots of different reasons.
+First and most commonly, user requests to GOV.UK may time out or return an error response.
 
-This is why we refer to switching off Nginx on the origin cache machines as
-"falling back to the mirrors".
+Second, if Fastly goes down, user requests to Fastly will fail with a 5xx error code (the problem was caused by the server). GOV.UK servers will still be able to serve content to users, but user requests will not be able to get to those servers.
```

Comment:
Small style thing, but should this be a numbered list (could help with scanning) and the sentence above ending with a `:`?

Reply:
In terms of style, we use numbered steps to guide a user through a process. I agree with your point re scanning, so perhaps an unordered list would help in this situation i.e.

"However, user requests will not get to Fastly in the following scenarios:

- first and most commonly, user requests to GOV.UK may time out or return an error response
- second, if Fastly goes down, user requests to Fastly will fail with a 5xx error code (the problem was caused by the server)"

@nicholsj how does that sound?